api/specs.proto

```diff
@@ -333,6 +333,12 @@ message ContainerSpec {
 	//
 	// https://docs.docker.com/engine/reference/commandline/run/#configure-namespaced-kernel-parameters-sysctls-at-runtime
 	map<string, string> sysctls = 26;
+
+	// Swap limit equal to memory plus swap: '-1' to enable unlimited swap
+	int64 memory_swap = 27;
+
+	// Tune container memory swappiness (0 to 100) (default -1)
+	google.protobuf.Int64Value memory_swappiness = 28;
```

Comment:
why 0 to 100? nit: maybe add a comment?

Reply:
also, curious why Int64Value instead of int64?

Reply:
Lazily copied from `docker run`'s help message: `--memory-swappiness int          Tune container memory swappiness (0 to 100) (default -1)` :)

Reply:
As to `Int64Value`, stayed consistent with https://github.com/moby/moby/blob/03e089e169a2889f096aceac6ad4a4e2ed6e186b/api/types/container/host_config.go#L333-L334 .

Reply:
what's the difference between `google.protobuf.Int64Value` and the `int64` above it? Does it make this compile to `*int64` in practice?

Reply:
Yes. It allows to differentiate between a missing value and a default one. Otherwise when deserializing the resulting struct will have a `0` value.

Reply:
what does (default -1) mean? Where is the default set? @wk8

Reply:
It might make sense to specify that the range 0-100 is specified in the linux kernel.

Reply:
@anshulpundir "what does (default -1) mean? Where is the default set?" => I guess somewhere deeper in moby?

Reply:
Does it make sense to specify that default value here then? What does the default value mean mean to the API consumer?

Reply:
Updated